### ISSUE TITLE
E22-900M30S output power configuration

### ISSUE THREAD

**Issue by SlitheringMule** [chandrawi/LoRaRF-Python#15]
I saw you mentioned that this library has been confirmed to work on the E22-900M30S, what output power did you use for the SX1262? There is a PA (power amplifier) which so outputting 22dBm may damage the board. What setting did you use? Thanks

**Comment by chandrawi** [comment]
Yes, this library works on E22-900M30S. +22 dBm output power doesn't damage the LoRa chip. 

To get +22 dBm you must add code like following
```cpp
LoRa.setTxPower(22) 
```
It will automatically set to the correct PA configuration for -22 dBm output power.

**Comment by chandrawi** [comment]
Yes, this library works on E22-900M30S. +22 dBm output power doesn't damage the LoRa chip. 

To get +22 dBm you must add code like following
```cpp
LoRa.setTxPower(22) 
```
It will automatically set to the correct PA configuration for -22 dBm output power.

**Comment by SlitheringMule** [comment]
I think 22dBm will damage the module as the Y2233W max input power is either 10dBm or 20dBm
